src/update_agent/mod.rs

```diff
@@ -13,7 +13,9 @@ use prometheus::IntGauge;
 use std::time::Duration;
 
 /// Default tick/refresh period for the state machine (in seconds).
-const DEFAULT_REFRESH_PERIOD_SECS: u64 = 5 * 60;
+const DEFAULT_REFRESH_PERIOD_SECS: u64 = 300; // 5 minutes.
```

Comment:
Is this used anywhere now?

Reply:
Yes, for non-steady states, here: https://github.com/coreos/zincati/blob/70fe933c63dac5492fb18bd2cf6679c3a0074c4d/src/update_agent/actor.rs#L115

Reply:
Ahh heh, I had grepped my local copy of Zincati, but hadn't `git pull`ed the latest tree. :)

Reply:
This reminds me I should enable `clippy` here, which would block the PR on unused items.

Reply:
`clippy` followup at https://github.com/coreos/zincati/pull/220.